test/modules/helpers.spec.ts

```diff
@@ -799,7 +799,8 @@ describe('helpers', () => {
         });
 
         it('should never return the callback result when probability is 0', () => {
-          const actual = faker.helpers.maybe(() => expect.fail(), {
+          const method = expect.fail as () => string;
```

Comment:
I have reservations about the use of type assertions Type assertions can inadvertently lead to bugs by causing a discrepancy between the expected types at runtime and during development. The current assertion seems misleading in the particularly context where `expect.fail` is actually typed to return `never` rather than `string`. 

Furthermore, considering that this project is open source, it is imperative to avoid employing workarounds that may obscure the true intent of the code. Instead of resorting to tricks for TypeScript/Linter compliance, I propose suggest `unknown` as the return type. This approach would not only alleviate the linter's confusion but also remain technically accurate without compromising the code's integrity.

```suggestion
          const method: () => unknown = expect.fail;
```

I'm sorry if this seems super nitpicky, but it's a concern important to me.

Reply:
Good catch. I like feedback like this as my coding style/habits are heavily influenced by Java.

Fixed